pages.nl/common/pio.md

```diff
@@ -0,0 +1,17 @@
+# pio
+
+> Ontwikkelomgeving voor voor embedded boards.
+> Sommige subcommando's zoals `pio run` hebben hun eigen documentatie.
+> Meer informatie: <https://docs.platformio.org/en/latest/core/userguide/>.
+
+- Toon help en toon subcommando's:
```

Comment:
```suggestion
- Toon help en toon subcommandos:
```

If apostrophies are used in Dutch as they are in English and German, this should be changed. However, please let me know if I'm wrong on  this one.